cloud/azure/modules/app/variables.tf

```diff
@@ -109,6 +109,11 @@ variable "log_retention" {
 }
 
 variable "ses_sender_email" {
-  type = string
+  type        = string
   description = "Email address of who is sending the email, passed to the app"
 }
+
+variable "staging_hostname" {
+  type        = string
+  description = "URL for staging"
```

Comment:
suggestion (blocking): please clarify URL or hostname. is the protocol included?